gulpjs/cloudbuild.yaml

```diff
@@ -0,0 +1,32 @@
+# In this directory, run the following command to build this builder.
+# $ gcloud container builds submit . --config=cloudbuild.yaml
+
+steps:
+# Step 0
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/gulpjs', '.']
+  dir: 'gulpjs'
+
+# Step 1
+# Print gulp version.
+- name: 'gcr.io/$PROJECT_ID/gulpjs'
+  args: ['--version']
+
+# Step 2
+# Gulp --help
+- name: 'gcr.io/$PROJECT_ID/gulpjs'
+  args: ['--help']
+
+# Step 3
+# Install gulp into the example build project
+- name: 'gcr.io/cloud-builders/npm'
+  args: ['install']
+  dir: 'gulpjs/examples/build'
+
+# Step 4
+# Gulp build
+- name: 'gcr.io/$PROJECT_ID/gulpjs'
+  args: ['build']
+  dir: 'gulpjs/examples/build'
```

Comment:
That would be nice to have an example that produces a static file, and in a next step pushes it to a CDN or to GCS.
I believe that's a realistic use case of gulpjs?